tensorflow_addons/optimizers/tests/stochastic_weight_averaging_test.py

```diff
@@ -72,15 +72,31 @@ def test_get_config():
     assert config["start_averaging"] == 0
 
 
-def test_assign_batchnorm():
+def test_mixed_precision():
+    tf.keras.mixed_precision.set_global_policy("mixed_float16")
```

Comment:
We have a pytest fixture

https://github.com/tensorflow/addons/blob/08767330c70ccd0974beaf3bb804dc565e9c9406/tensorflow_addons/utils/test_utils.py#L111